scripts/populate_conferences.py

```diff
@@ -261,6 +261,14 @@ def main():
         'admins': [],
         'public_projects': True,
     },
+        'OSFM2015': {
```

Comment:
Whitespace is significant in Python, so this should actually be outdented. I'll make this change when I merge.